packages/checkbox/src/CheckboxPresenter/CheckboxPresenter.test.js

```diff
@@ -1,10 +1,15 @@
 import React from "react";
 import renderer from "react-test-renderer";
 import CheckboxPresenter from "./CheckboxPresenter";
+import { generateId } from "@hig/utils";
 
 describe("Checkbox/CheckboxPresenter/CheckboxPresenter", () => {
+  afterEach(() => {
+    generateId.mockReset();
+  });
+
   [
-    {
```

Comment:
This is causing only one snapshot to be captured.

Reply:
Nice one, updated in 404046b4